src/front-end/src/helper-methods.js

```diff
@@ -2,6 +2,29 @@ import firebase from './firebase-init';
 
 const auth = firebase.auth();
 
+export function resendVerificationEmail(user) {
+  user
+    .sendEmailVerification()
+    .then(() => {
+      window.alert(
+        'An new verification email sent has successfully been sent!',
+      );
+      console.log('Success');
+    })
+    .catch((error) => {
+      console.log(error);
+      switch (error.code) {
+        case 'auth/too-many-requests':
+          window.alert(
+            'Please check the inbox and junk folder of your email before pressing the button again',
+          );
+          break;
+        default:
+          break;
+      }
+    });
+}
+
 function mySendEmailVerification(user) {
```

Comment:
Please note: I'm aware that resendVerificationEmail function is essentially the same as the mySendEmailVerification function. I did try to export mySendEmailVerification instead, however, the window alerts weren't being displayed correctly (despite my best efforts, such as using try catch statements). As such, I made the resendVerificationEmail function since the mySendEmailVerification is used to send the verification to admins and mentors upon signup (and thus do not need the popups like the resendVerificationEmail function does)